### PR TITLE
Add FAQ item for troubleshooting potential rclone error

### PR DESCRIPTION
**Pull request by d33bs**
This PR adds documentation to help troubleshoot potential scenarios where data upload is prevented based on the bucket settings and how rclone operates.

### CODE REVIEW

File: utilities/durbinlab/README.md [hunk 1]
```diff
@@ -17,3 +17,10 @@ We suggest using [`rclone`](https://rclone.org) to assist with data uploads to t
 
    - Please note: many rclone commands are recursive __by default__ with options to disable.
    - An example with the sync command: `rclone sync <data source>  <configured_name>:waylab-durbinlab-bucket`
+
+## Frequently Asked Questions (FAQ)
+
+- __Question__: What should I do if I see the error: `"Error 400: Cannot insert legacy ACL for an object when uniform bucket-level access is enabled."`?
+- __Answer__: Try using the following:
+  - Add an environment variable like the following (command may vary based on operating system): `export RCLONE_GCS_BUCKET_POLICY_ONLY=true`
+  - Use an additional command-line flag `rclone --gcs-bucket-policy-only ...<the rest of your command goes here>...`
```
Comment (falquaddoomi) [inline utilities/durbinlab/README.md:26]: My understanding from the docs (https://rclone.org/googlecloudstorage/#gcs-bucket-policy-only) is that you just need one of those options: specifying `--gcs-bucket-policy-only` is the same as setting `RCLONE_GCS_BUCKET_POLICY_ONLY=true` and as far as I can tell the flag overrides the env var. Maybe saying something like "Try using one of the following options" would be clearer?

**Review by falquaddoomi — COMMENTED** [review]
Looks good to me! I left a minor comment about clarifying that you just need one of the two options, but otherwise it looks good to merge IMHO.

File: utilities/durbinlab/README.md (hunk 1 above)
Comment (d33bs) [reply utilities/durbinlab/README.md:26]: Thanks, great point, I'll get this updated!

**Review by falquaddoomi — APPROVED** [review]

**Review by gwaybio — APPROVED** [review]
LGTM

General question (no need to delay merging) - this is forked from `gc-assayworks-bucket`, but why not spawn from the https://github.com/CU-DBMI/gc-cloud-storage-bucket template?

**Comment by d33bs** [comment]
Thanks @gwaybio and @falquaddoomi !

> General question (no need to delay merging) - this is forked from gc-assayworks-bucket, but why not spawn from the https://github.com/CU-DBMI/gc-cloud-storage-bucket template?

I felt that `gc-assayworks-bucket` was more up to date at the time (there were changes which I believe only took place in the `assayworks` repo after the original fork). We should port the changes which occurred there and here to the `CU-DBMI/gc-cloud-storage-bucket` to help retain the information in one spot (while respecting the differences).

**Pull request merged**